tools/bazel.rc

```diff
@@ -12,6 +12,7 @@ startup --batch_cpu_scheduling
 
 startup --host_jvm_args=-XX:-UseParallelGC
 
+startup --output_user_root=/apollo/bazel-root
```

Comment:
Please also retire the old cache mechanism, such as:
https://github.com/ApolloAuto/apollo/blob/master/docker/scripts/dev_start.sh#L211
https://github.com/ApolloAuto/apollo/blob/master/docker/scripts/dev_start.sh#L283

And as it's a big change, please conduct intensive testing to verify that everything works as expected :)

Reply:
Good catch. It's important to let the team test internally to an extensive extent before pushing to the public.

Reply:
> Good catch. It's important to let the team test internally to an extensive extent before pushing to the public.

Got it. Actually I tested on bazel-0.5.3 and bazel-3.x systems. But didn't realize  that `apollo.sh` script have that directory mounted.  Thanks Xiangquan for pointing it out.